l10n_pt_vat/__openerp__.py

```diff
@@ -21,18 +21,18 @@
 
 {
     "name": "Portugal - IVA",
-    "version": "8.0.0.0.2",
+    "version": "8.0.0.0.3",
     "license": "AGPL-3",
     "depends": ['account', 'base_vat'],
     "author": "Sossia, Odoo Community Association (OCA)",
     "summary": "Portuguese VAT requirements extensions",
-    "website": "http://www.sossia.pt",
-    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/l10n-portugal",
+    "category": "Localisation/Europe",
```

Comment:
OK. I did some research and there's no fixed rule for that.
I suggest we use "Localization/Portugal".
At least Italy does do, and IMHO I think it's better than "Localisation/Europe".
BTW I think that "Localization" is spelled with a "z".